types/noloopfor/tslint.json

```
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+      "whitespace": false
+  }
```

```suggestion
  "extends": "dtslint/dt.json"
```
why do you need this exclusion?
https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson